xml/System.Threading/WaitCallback.xml

```diff
@@ -45,40 +45,17 @@
 ## Remarks  
  <xref:System.Threading.WaitCallback> represents a callback method that you want to execute on a <xref:System.Threading.ThreadPool> thread. Create the delegate by passing your callback method to the <xref:System.Threading.WaitCallback> constructor. Your method must have the signature shown here.  
   
- Queue your task for execution by passing the <xref:System.Threading.WaitCallback> delegate to <xref:System.Threading.ThreadPool.QueueUserWorkItem%2A?displayProperty=nameWithType>. Your callback method executes when a thread pool thread becomes available.  
+ Queue the method for execution by passing the <xref:System.Threading.WaitCallback> delegate to <xref:System.Threading.ThreadPool.QueueUserWorkItem%2A?displayProperty=nameWithType>. The callback method executes when a thread pool thread becomes available.  
   
 > [!NOTE]
 >  Visual Basic users can omit the <xref:System.Threading.WaitCallback> constructor, and simply use the `AddressOf` operator when passing the callback method to <xref:System.Threading.ThreadPool.QueueUserWorkItem%2A>. Visual Basic automatically calls the correct delegate constructor.  
   
- If you want to pass information to your callback method, create an object that contains the necessary information and pass it to <xref:System.Threading.ThreadPool.QueueUserWorkItem%2A> when you queue your task for execution. Each time your callback method executes, the `state` parameter contains this object.  
+ If you want to pass information to the callback method, create an object that contains the necessary information and pass it to the <xref:System.Threading.ThreadPool.QueueUserWorkItem(System.Threading.WaitCallback,System.Object)> method as the second argument. Each time the callback method executes, the `state` parameter contains this object.  
   
- For more information about using callbacks to synchronize thread pool threads, see [The Managed Thread Pool](~/docs/standard/threading/the-managed-thread-pool.md).  
-  
-   
-  
-## Examples  
```

Comment:
Although removing the example is fine, it would be good to mention and link to the examples: "For examples that use the \`WaitCallback\` delegate, see the \<xref:System.Threading.ThreadPool.QueueUserWorkItem%2A?displayProperty=nameWithType> method."

Reply:
@rpetrusha that's a good remark. I've added that sentence to the docs. Thank you for review.